app/src/main/java/com/duckduckgo/app/global/DuckDuckGoActivity.kt

```diff
@@ -75,6 +77,12 @@ abstract class DuckDuckGoActivity : AppCompatActivity() {
         super.onDestroy()
     }
 
+    fun setupToolbar(toolbar: Toolbar) {
```

Comment:
because there's a new Back icon and this is called from 4 different Activities, we've created a new method in the base Activity